_specs/ecip-1000.md

```diff
@@ -353,6 +355,19 @@ In addition, it is recommended that literal code included in the ECIP be dual-li
 * Some ECIPs, especially consensus layer, may include literal code in the ECIP itself which may not be available under the exact license terms of the ECIP.
 * Despite this, not all software licenses would be acceptable for content included in ECIPs.
 
+# ECIP Process Code of Conduct
+
+## ECIP Participants Will Adhere to the Following
+
+1. Anyone can participate in the ECIP process as a competent developer, miner, validator, node operator, user, or any other prescribed participant or stakeholder.
+2. Within the ECIP documents and discussion pages, participants will conduct themselves professionally, focusing on ECIP issues, making sure their arguments are well founded, minimizing trolling, insulting or derogatory comments, and personal or political attacks.
```

Comment:
I like the word choice of saying "minimizing" since that's the most realistic thing to expect in an open-system that maintains permissionlessness while not commiting the permissionless fallacy.